packages/xod-tabtest/src/TabData.re

```diff
@@ -62,6 +63,12 @@ module Value = {
       byteStringToInt(byteString)
     | x => Invalid(x)
     };
+  let parseTime = str =>
+    switch (str) {
+    | numString when Re.test(numString, numberRegex) =>
```

Comment:
I think using `numberRegex` is not correct. We should accept positive integers only

Reply:
Fixed.